app/scripts/directives/bindService.js

```diff
@@ -178,10 +180,13 @@
       });
 
       if (ctrl.target.kind === 'Instance') {
-        ctrl.bindType = "secret-only";
-        ctrl.appToBind = null;
-        ctrl.serviceToBind = ctrl.target.metadata.name;
-        loadApplications();
+        ProjectsService.get(ctrl.target.metadata.namespace).then(_.spread(function(project) {
```

Comment:
You should update the bindService component to take the project as a param, anywhere we are using that component the project has already been loaded by the main controller for the page